### ISSUE TITLE
Position-only separator in first parameter should be flagged as error

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1064]
```python
def foo(/, a):
   pass
```

The Python interpreter emits a syntax error in this case. Pyright should therefore report an error.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.75, which I just published. It will also be included in the next version of Pylance.
